### PR TITLE
Submit pwkit

### PR DESCRIPTION
**Pull request by pkgw**

### CODE REVIEW

File: recipes/pwkit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  entry_points:
+    - astrotool = pwkit.cli.astrotool:commandline
+    - casatask = pwkit.environments.casa.tasks:commandline
+    - imtool = pwkit.cli.imtool:commandline
+    - pkcasascript = pwkit.environments.casa.scripting:commandline
+    - pkenvtool = pwkit.environments:commandline
+    - wrapout = pwkit.cli.wrapout:commandline
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/pwkit/meta.yaml]: It looks like only `python` and `setuptools` are `build` requirements.

File: recipes/pwkit/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    - setuptools
+    - six
+
+  run:
+    - numpy
+    - pathlib  # [py2k]
+    - python
+    - six
+
+test:
```
Comment (synapticarbors) [inline recipes/pwkit/meta.yaml]: Can you add tests for the `entry_points`:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

Comment (pkgw) [reply recipes/pwkit/meta.yaml]: Done, skipping the CASA-related ones, which require a very large software stack to be installed in order to run.

**Comment by pkgw** [comment]
OK, hopefully this is now good to merge.

**Pull request merged**